doc/architecture/adr/0019-message-broker.md

```diff
@@ -0,0 +1,53 @@
+Message broker
+
+* Status:
+* Deciders: @anna-knudsen, @duizer, @ckr123, @Exobitt
+* Date: 2022-01-19
+
+---
+
+## Context and Problem Statement
+
+For our event driven architecture solution we need a message broker that can handle a large number of messages.
+We also want a message broker that is fairly easy to set up.
+
+---
+
+## Considered Options
+
+* Kafka
+* RabbitMQ
+* Redis
+* ActiveMQ
+
+---
+
+## Decision Outcome
+
+We are going with RabbitMQ
+
+## Rationale
+
+RabbitMQ seems to be a simple message broker and not much else, which is exactly what we need.
+It is a very popular, open source message broker with lots of support and documentation.
+It also seems very easy to set up.
+
+Kafka:
+We chose not to go with Kafka, as it seems very complex - it is more like a streaming service and provides much more functionality than we need.
+We get the impression that Kafka requires experts to fine tune after the initial setup
+
+ActiveMQ:
+Active MQ is mostly for Java.
+
+Redis:
+Redis doesn't have built-in SSL and offers no persistence out of the box.
+
+### Positive Consequences
+
+Easy setup
+Lots of documentation
+
+### Negative Consequences
+
+We are not yet 100% sure that RabbitMQ can handle the number of messages we need, and we are aware that RabbitMQ can not be scaled to the extent that Kafka can.
```

Comment:
Is it possible to add the number of messages that we currently think that we need?